lib/badge.js

```diff
@@ -119,8 +119,8 @@ function makeImage(data, cb) {
   if (textWidth1 % 2 === 0) { textWidth1++; }
   if (textWidth2 % 2 === 0) { textWidth2++; }
   data.widths = [
-    textWidth1 + 10 + data.logoWidth + data.logoPadding,
-    textWidth2 + 10,
+    textWidth1 + 20 + data.logoWidth + data.logoPadding,
+    textWidth2 + 30,
```

Comment:
it was causing overlap on longer test strings from the description... if you pull it and run it with old values you'll see what i mean.